packages/docz-tools/gatsby-node.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-env node */
+const path = require("path");
+
+const { USE_ATLANTIS_ALIASES } = process.env;
+const useAtlantisAliases = USE_ATLANTIS_ALIASES == "true";
```

Comment:
Did you use `==` for a reason here?

```suggestion
const useAtlantisAliases = USE_ATLANTIS_ALIASES === "true";
```